terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/FunctionTestSupport.java

```diff
@@ -182,6 +182,7 @@ private WebDriver newWebDriver() {
     protected void quitDefaultWebDriver() {
         if (driver != null) {
             try {
+                driver.get("about:config");
```

Comment:
- [ ] Why need this? Please re-consider based #768.

Reply:
Confirmation of ABC has leaked.
I deleted in the same way because it has been deleted in ABC.
54bd477db78b89598512792318284aaed9458bc5